roles/edpm_libvirt/defaults/main.yml

```diff
@@ -62,7 +62,11 @@ edpm_libvirt_packages:
   # ceph-common is needed for ceph support to provide the ceph client lib
   # for qemu and the ceph udev rules
   - ceph-common
+  # for SASL auth
+  - cyrus-sasl-scram
 edpm_libvirt_ceph_path: /var/lib/openstack/config/ceph
+edpm_libvirt_password_path: /var/lib/openstack/configs/{{ edpm_service_name | default('libvirt') }}/LibvirtPassword
```

Comment:
does this really need to be configurable?

when would we ever set edpm_service_name to something other the libvirt.

unlike nova we do not intend to allow custom libvirt dataplane services as we do not want to allow arbitrary configuration ro libvirt.

Reply:
you must support custom libvirt services at least to allow a different issuer to be use for the TLS certs

Reply:
that not something we planned to do

we could but that is beyond the MVP we require
I'm not against you supporting it in the code but I'm not a fan of documenting this
in our product docs.

i do not want to suggest to customers that we intend to allow them to modify the libvirt configuration in general because we don't want to support that.

Reply:
maybe not have a choice on this if we want to reach feature parity e.g setting the listen address to a specific network